org.gecko.bnd.eclipse.launcher.pre/bnd.bnd

```diff
@@ -1,6 +1,6 @@
+-library: enable-emf
```

Comment:
Why do we need EMF here? The Eclipse lauchner should not have a dependency to EMF